{
  "name": "DataDirectoryException",
  "comment": "DataDirectoryException is thrown when a folder item can not be \n created because its associated data directory already exists.",
  "javadoc": "\u003ccode\u003eDataDirectoryException\u003c/code\u003e is thrown when a folder item can not be \n created because its associated data directory already exists.",
  "static": false,
  "implements": [],
  "extends": "java.io.IOException",
  "fields": [
    {
      "name": "dir",
      "comment": "",
      "javadoc": "",
      "static": false,
      "type_long": "java.io.File",
      "type_short": "File",
      "constant_value": null
    }
  ],
  "methods": [
    {
      "name": "\u003cinit\u003e",
      "comment": "Constructor.",
      "javadoc": "Constructor.\n@param msg error message\n@param dir existing data directory",
      "static": false,
      "params": [
        {
          "name": "msg",
          "type_long": "java.lang.String",
          "type_short": "String",
          "comment": "error message"
        },
        {
          "name": "dir",
          "type_long": "java.io.File",
          "type_short": "File",
          "comment": "existing data directory"
        }
      ],
      "return": {
        "type_long": "void",
        "type_short": "void",
        "comment": ""
      },
      "throws": []
    },
    {
      "name": "getDataDirectory",
      "comment": "Returns existing data directory",
      "javadoc": "Returns existing data directory",
      "static": false,
      "params": [],
      "return": {
        "type_long": "java.io.File",
        "type_short": "File",
        "comment": ""
      },
      "throws": []
    }
  ]
}
